src/components/AddCourseModal.tsx

```diff
@@ -0,0 +1,108 @@
+import React, { useContext, useState } from 'react';
+import styled, { ThemeContext } from 'styled-components';
+import DefaultModal from './DefaultModal';
+import Logo from '../molecules/Logo';
+import TextInput from './TextInput';
+
+const Body = styled.div`
+	height: 100%;
+	width: 100%;
+	display: grid;
+	grid-template-rows: 1fr 4fr 5fr;
+	grid-template-areas:
+		'desc'
+		'logo'
+		'inputs';
+	justify-items: center;
+`;
+
+const Desc = styled.p`
+	font-size: ${({ theme }) => theme.fontSizes.xs};
+`;
+
+const InputField = styled.div`
+	margin: 0 2.2em 0 2em;
+	display: block;
+	justify-content: center;
+	align-items: center;
+`;
+
+const InputName = styled(TextInput)``;
```

Comment:
Hey Seiya, is this needed? Excited that we have an add course modal now!!

Reply:
Oh I forgot to put the padding on this component, thanks for the catch! Also took out line 27 since it wasn't needed.